docker/test/Dockerfile

```diff
@@ -27,6 +27,15 @@ RUN wget --no-check-certificate -q https://github.com/protocolbuffers/protobuf/a
     make install && ldconfig && \
     rm -rf /protobuf-3.9.0
 
+# JSON for Modern C++
+
+RUN wget --no-check-certificate -q https://github.com/nlohmann/json/archive/refs/tags/v3.10.2.zip && \
```

Comment:
Luis will answer that but my main motivation to use it instead of jsoncpp is that nlohmann is under active development while jsoncpp seems to be in maintenance mode.

Reply:
yep, we are using libcomm and the c++ client to test this (header-only) json library that seems more up-to-date and modern. At some point, we can update aperturedb's too.